NoteFind/View Controllers/NewNoteViewController.h

```diff
@@ -7,11 +7,18 @@
 
 #import <UIKit/UIKit.h>
 #import "Tags.h"
+#import "TagsViewController.h"
```

Comment:
Given there are no actually code changes in this header, we should move these imports to the implementation (.m) file. This actually helps improve the build speed of your app since any place that imports this header (.h) file will automatically also import any files listed in this file

Reply:
wow! i didn't know this. i assumed everything was supposed to be imported to .h file for a cleaner look